Dispatchfile

```diff
@@ -1,7 +1,58 @@
 #!mesosphere/dispatch-starlark:v0.6
 # vi:syntax=python
+
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.4", "gitResource", "pullRequest", "push", "volume", "resourceVar", "secretVar")
 load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.7", "cron")
+load("github.com/mesosphere/cluster-claim-controller/starlark/claim@master", "make_kind_cluster", "fetch_kubeconfig", "cleanup")
 load("github.com/mesosphere/dispatch-tasks/bump_charts/bump_charts@master", "bump_charts")
 
+
 do_bump_charts = bump_charts(repo_name="kubeaddons-enterprise", task_name="bump-charts")
+
+git = "src-git"
+gitResource(git, url="$(context.git.url)", revision="$(context.git.commit)")
+
+cluster_name = "kubeaddons"
+kind_cluster=make_kind_cluster(cluster_name, git)
+
+task("integration-test", inputs=[git],
+     deps=[kind_cluster],
+     steps=[
+       fetch_kubeconfig(cluster_name, git),
+       k8s.corev1.Container(
+       name = "integration-tester",
+              workingDir = "/workspace/{}/".format(git),
+              image = "mesosphere/kubeaddons-ci:latest",
+              command = ["/bin/bash", "-c"],
+              args = [
+                  """
+                  set -euo pipefail
+                  export GIT_TERMINAL_PROMPT=1
+                  export GOPRIVATE=github.com/mesosphere/kubeaddons
+                  ssh-keyscan -t rsa github.com > /etc/ssh/known_hosts
+                  git config --global url."https://$GITHUB_TOKEN:@github.com/".insteadOf "https://github.com/"
+                  make dispatch-test
+                  """
+              ],
+              resources = k8s.corev1.ResourceRequirements(
+              limits = {
+                "cpu": k8s.resource_quantity("8000m"),
+                "memory": k8s.resource_quantity("6Gi")
+              },
+              requests = {
+                "cpu": k8s.resource_quantity("1000m"),
```

Comment:
You can probably request a lot less than this maybe 500m

Reply:
Updated.